include/RAJA/exec-cuda/reduce_cuda.hxx

```diff
@@ -607,13 +571,7 @@ public:
 
       if (threadIdx.x < 1) {
           sd[0] = RAJA_MAX(sd[0],sd[1]);
-#if defined(RAJA_USE_NO_ATOMICS)
-          m_blockdata[m_blockoffset + blockIdx.x+1]  =
-              RAJA_MAX( sd[0], m_blockdata[m_blockoffset + blockIdx.x+1] );
-
-#else
-          atomicMax( &m_blockdata[m_blockoffset], sd[0] );
-#endif
+          atomicMax(&(m_tallydata->tally), sd[0] );
```

Comment:
Note that the argument to this class is cuda_reduce<>, and not cuda_reduce_atomic<>.  The ReduceMin class implemented immediately above this class is implemented correctly.  This class should look like the class immediately above it.